blockchain/message_pool/src/msgpool.rs

```diff
@@ -939,6 +1353,11 @@ pub mod test_provider {
                     }
                 }
             }
+            let balance = match self.balances.get(addr) {
+                Some(b) => b.clone(),
+                None => (10_000_000_000 as u64).into(),
```

Comment:
if this is intentional, can you at least document where this is coming from? Seems like it should be coming from a protocol constant

Reply:
its just for testing